test/SourceKit/Refactoring/basic.swift

```diff
@@ -198,12 +206,12 @@ func `hasBackticks`(`x`: Int) {}
 // CHECK-GLOBAL: ACTIONS BEGIN
 // CHECK-GLOBAL-NEXT: source.refactoring.kind.rename.global
 // CHECK-GLOBAL-NEXT: Global Rename
-// CHECK-GLOBAL-NEXT: ACTIONS END
+// CHECK-GLOBAL: ACTIONS END
```

Comment:
I'm thinking the intention here could have been to check that it gave global rename and not local rename. Maybe put a `// CHECK-GLOBAL-NOT: Local Rename` above and below the global match and drop the first `-NEXT`.